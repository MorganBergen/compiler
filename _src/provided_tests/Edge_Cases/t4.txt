TASKS
    2 5
POLY
    F(X) = X + 1;
    G(Y) = F(Y)^2;
EXECUTE
    INPUT A;
    A = G(4);
    OUTPUT A;
INPUTS
    0
