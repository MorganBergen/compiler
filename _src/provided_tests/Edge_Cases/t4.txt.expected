25
F: 1
G: 2
